<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<title>Question 1 test html</title>
    
    <script src="redhat-q1.js"></script>    
    <script type="text/javascript">
        setTimeout(foo(function(err, res){
 console.log('Done!. err='+err+ ' : res = '+ res)
}), 2000);
        
    </script>    
    
</head>
<body>
    <p>This is Test HTML for question 1 given in part2. Before fixing, callback function was calling twice if error has been reported from doThing method. That is fixed. To test that, this HTML submitted asynchrnous call to foo function to see callback function is getting called only once based on doThing request.</p>
</body>
</html>